test/v3/priceImpact/priceImpact.V3.integration.test.ts

```diff
@@ -176,7 +177,8 @@ describe('PriceImpact V3', () => {
         });
     });
 
-    describe('Nested pool', () => {
+    // FIXME: zeroOutDeltas is swapping a huge amount (~200 WETH) and hitting MaxInRatio
+    describe.skip('Nested pool', () => {
```

Comment:
This one is indeed not working, but I'd suggest we fix this in a separate PR, so we unblock the deploy update. Let me know what you think and I'll create a ticker over here

Reply:
I agree with kicking this down the road